.clang-format

```diff
@@ -1,5 +1,7 @@
 AlwaysBreakTemplateDeclarations: true
 BreakBeforeBraces: Allman
```

Comment:
I like these changes, but in the future we should think about making changes like this in their own PR's.